pom.xml

```diff
@@ -674,6 +674,18 @@
 				<tests.to.skip>**/*OnlineTest.java</tests.to.skip>
 			</properties>
 		</profile>
+		<profile>
+		    <id>skip-all-tests</id>
+		    <activation>
+			<property>
+			    <name>skip-all-tests</name>
+			    <value>true</value>
+			</property>
+		    </activation>
+		    <properties>
+			<maven.test.skip>true</maven.test.skip>
+		    </properties>
+		</profile>
```

Comment:
What does this change do?

Reply:
It adds a profile in NetBeans for skipping all tests (rather than the existing “skip-online-tests” which only skips the network requiring tests)